### PR TITLE
[Accessibility] Keyboard navigation in the sideDoc

### PR DESCRIPTION
**Pull request by veler**
- When the side docs is opening, the focus is given to the Expand/Collapse button.
- Made the PXT buttons accessible into the documentation that appears in the side doc.

Related issue : [https://github.com/Microsoft/pxt/issues/2527](https://github.com/Microsoft/pxt/issues/2527), [https://github.com/Microsoft/pxt/issues/2530](https://github.com/Microsoft/pxt/issues/2530), [https://github.com/Microsoft/pxt/issues/2529](https://github.com/Microsoft/pxt/issues/2529)

### CODE REVIEW

File: pxtrunner/renderer.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ namespace pxt.runner {
 
             // js menu
             if ($svg) {
-                const $svgBtn = $('<a class="item blocks"><i aria-label="Blocks" class="puzzle icon"></i></a>').click(() => {
+                const $svgBtn = $('<a class="item blocks" role="button" tabindex="0" aria-label="Blocks"><i role="presentation" aria-hidden="true" class="puzzle icon"></i></a>').click(() => {
```
Comment (microsoftsam) [inline pxtrunner/renderer.ts]: Wrap Blocks around lf.

Comment (veler) [inline pxtrunner/renderer.ts]: Oops, I probably though I was on pxt-microbit at this moment.

File: webapp/src/container.tsx [hunk 2]
```diff
@@ -98,7 +107,7 @@ export class SideDocs extends data.Component<ISettingsProps, {}> {
         if (!docsUrl) return null;
 
         return <div>
-            <button id="sidedocstoggle" role="button" aria-label={state.sideDocsCollapsed ? lf("Expand the side documentation") : lf("Collapse the side documentation")} className="ui icon button" onClick={() => this.toggleVisibility() }>
+            <button id="sidedocstoggle" role="button" aria-label={state.sideDocsCollapsed ? lf("Expand the side documentation") : lf("Collapse the side documentation. Press tab to navigate into the documentation.")} className="ui icon button" onClick={() => this.toggleVisibility() }>
```
Comment (microsoftsam) [inline webapp/src/container.tsx]: Not sure this is necessary, I think it should be obvious that tabbing is going to navigate.

Comment (veler) [inline webapp/src/container.tsx]: Hum right.

**Pull request merged**